src/Docker.DotNet/Models/GlobalJob.Generated.cs

```diff
@@ -0,0 +1,9 @@
+using System.Runtime.Serialization;
+
+namespace Docker.DotNet.Models
+{
+    [DataContract]
+    public class GlobalJob // (swarm.GlobalJob)
+    {
```

Comment:
Odd this has no members?

Reply:
https://godoc.org/github.com/docker/docker/api/types/swarm#GlobalJob

> GlobalJob is the type of a Service which executes a Task on every Node matching the Service's placement constraints. These tasks run to completion and then exit.
This type is deliberately empty.

Reply:
Lol. Thanks